src/providers/consensus/typings.py

```diff
@@ -11,6 +11,24 @@ class BeaconSpecResponse(FromResponse):
     DEPOSIT_CHAIN_ID: str
     SLOTS_PER_EPOCH: str
     SECONDS_PER_SLOT: str
+    # https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#time-parameters-1
+    MIN_VALIDATOR_WITHDRAWABILITY_DELAY: str
```

Comment:
Maybe lets parse response to expected value

Reply:
We don't do it in other requests. Why should we do it here?